### ISSUE TITLE
[Request] Single tile widget to Freeze all Selected (Home) apps

### ISSUE THREAD

**Issue by ChaliceChore** [aistra0528/Hail#266]
Pretty much the title.
Please add such a widget to quickly freeze apps.

**Comment by Infiniti151** [comment]
Yes, this would make it very convenient. A single tile widget to freeze/unfreeze a particular tag group. Here is how the flow can be: A user selects the widget, the widget displays all the tag groups (Default and any custom ones the user has added), the user selects a tag group. Now the widget will freeze/unfreeze only that tag group.

**Comment by ChaliceChore** [comment]
It's been 2 years since I made that request so I don't think it is going to be implemented any time soon.

**Comment by Mykesss** [comment]
> It's been 2 years since I made that request so I don't think it is going to be implemented any time soon.

Widget/shortcut is there, in the app settings on the bottom.

**Comment by ChaliceChore** [comment]
> Widget/shortcut is there, in the app settings on the bottom.

Oh yeah, you are right. Thanks!

**Comment by Infiniti151** [comment]
> > It's been 2 years since I made that request so I don't think it is going to be implemented any time soon.
> 
> Widget/shortcut is there, in the app settings on the bottom.

That's cool! But there is still no single widget to toggle freeze/unfreeze tags. I need to create two different widgets to freeze and unfreeze a single tag which sucks.

**Comment by ChaliceChore** [comment]
> Il no single widget to toggle freeze/unfreeze tags. I need to create two different widgets to freeze and unfreeze a single tag which sucks.

I don't quite understand your grievance. Wouldn't a single widget with two-buttons (freeze/unfreeze) occupy same real estate on your launcher as two separate buttons?

**Comment by Infiniti151** [comment]
> > Il no single widget to toggle freeze/unfreeze tags. I need to create two different widgets to freeze and unfreeze a single tag which sucks.
> 
> I don't quite understand your grievance. Wouldn't a single widget with two-buttons (freeze/unfreeze) occupy same real estate on your launcher as two separate buttons?

Why would the single widget need two buttons? It is possible to create a single tile toggle widget with RemoteViews/JetPack Glance. The problem doesn't seem much for a single tag (2 tiles), but becomes worse for more tags. If someone has 4 tags, that means 8 tiles which is one whole extra row.

**Comment by ChaliceChore** [comment]
Aah I see. I didn't realize that, I just use the app like a simpleton.
